.github/workflows/lawa-js-ci.yml

```diff
@@ -26,6 +26,8 @@ on:
     secrets:
       github-token:
         required: true
+      sidekiq-enterprise-config:
```

Comment:
I don't think we need sidekiq here as bundler is only used to install the license checker I believe

Reply:
As per the [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callsecrets) if caller passes a secret and its not specified, it may raise an error.